sphinx/ext/intersphinx/_load.py

```diff
@@ -78,18 +69,18 @@ def validate_intersphinx_mapping(app: Sphinx, config: Config) -> None:
                 'Invalid value `%r` in intersphinx_mapping[%r]. '
                 'Expected a two-element tuple or list.'
             )
-            LOGGER.error(msg % (value, name))
+            LOGGER.error(msg, value, name)
             del config.intersphinx_mapping[name]
             continue
         try:
             uri, inv = value
-        except (TypeError, ValueError, Exception):
+        except Exception:
```

Comment:
Whilst Exception is the superclass, this gives specificity -- we would normally _expect_ Type or Value errors.
```suggestion
        except (TypeError, ValueError, Exception):
```

Reply:
Ah, I understand now why you did this. Ok for this one.